integration/benchnet2/automate/bootstrap_automation.go

```diff
@@ -0,0 +1,196 @@
+	if templatePath != "" {
+		templateFolder = templatePath
+	}
+	nodeInfoJson := DEFAULT_NODE_INFO_PATH
+	if inputJsonFilePath != "" {
+		nodeInfoJson = inputJsonFilePath
+	}
+	nodesData, nodeConfig := loadNodeJsonData(nodeInfoJson)
+
+	values := buildValuesStruct(templateFolder, nodesData, nodeConfig)
+
+	marshalToYaml(values, outputYamlFilePath)
+}
+
+// Loads node info json file and returns unmarshaled struct and node counts
+func loadNodeJsonData(nodeInfoPath string) (map[string]Node, map[string]int) {
+	jsonFileBytes := textReader(nodeInfoPath)
+
+	var nodes []Node
+	err := json.Unmarshal(jsonFileBytes, &nodes)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var nodeConfig = make(map[string]int)
+	nodeConfig["access"] = 0
+	nodeConfig["collection"] = 0
+	nodeConfig["consensus"] = 0
+	nodeConfig["execution"] = 0
+	nodeConfig["verification"] = 0
+
+	nodeMap := map[string]Node{}
+	nodeNameRe := regexp.MustCompile(`\w{6,}\d{1,3}`)
+	nodeTypeRe := regexp.MustCompile(`\D{6,}`)
+	for _, node := range nodes {
+		name := nodeNameRe.FindStringSubmatch(node.Address)
+		nodeType := nodeTypeRe.FindStringSubmatch(node.Address)
+		nodeMap[name[0]] = node
+		nodeConfig[nodeType[0]]++
+	}
+
+	return nodeMap, nodeConfig
+}
+
+func buildValuesStruct(templateFolder string, nodesData map[string]Node, nodeConfig map[string]int) *Values {
+	resources := string(textReader(templateFolder + RESOURCES_TEMPLATE))
+	nodeTypeResources := unmarshalToStruct(resources, &Defaults{}).(*Defaults)
+
+	accessTemplatePath := templateFolder + ACCESS_TEMPLATE
+	accessNodeMap := nodeStruct("access", accessTemplatePath, DEFAULT_ACCESS_IMAGE, nodesData, nodeConfig)
+	var accessNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: accessNodeMap}
+
+	collectionTemplatePath := templateFolder + COLLECTION_TEMPLATE
+	collectionNodeMap := nodeStruct("collection", collectionTemplatePath, DEFAULT_COLLECTION_IMAGE, nodesData, nodeConfig)
+	var collectionNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: collectionNodeMap}
+
+	consensusTemplatePath := templateFolder + CONSENSUS_TEMPLATE
+	consensusNodeMap := nodeStruct("consensus", consensusTemplatePath, DEFAULT_CONSENSUS_IMAGE, nodesData, nodeConfig)
+	var consensusNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: consensusNodeMap}
+
+	executionTemplatePath := templateFolder + EXECUTION_TEMPLATE
+	executionNodeMap := nodeStruct("execution", executionTemplatePath, DEFAULT_EXECUTION_IMAGE, nodesData, nodeConfig)
+	var executionNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: executionNodeMap}
+
+	verificationTemplatePath := templateFolder + VERIFICATION_TEMPLATE
+	verificationNodeMap := nodeStruct("verification", verificationTemplatePath, DEFAULT_VERIFICATION_IMAGE, nodesData, nodeConfig)
+	var verificationNodes = NodesDefs{Defaults: *nodeTypeResources, Nodes: verificationNodeMap}
+
+	return &Values{Branch: "fake-branch", Commit: "123456", Defaults: EmptyStruct{}, Access: accessNodes, Collection: collectionNodes,
```

Comment:
The `Branch` and `Commit` do not need to be hard coded in the return. Ideally, these should be passed through via CLI args.